sources/packages/backend/libs/services/src/sfas/sfas-individual.service.ts

```diff
@@ -38,4 +38,34 @@ export class SFASIndividualService extends DataModelService<SFASIndividual> {
 
     return individual?.pdStatus;
   }
+
+  /**
+   * Search for a record in SFAS table using student details.
+   * @param lastName student's last name.
+   * @param birthDate student's birth date.
+   * @param sin student's sin.
+   * @returns SFAS individual details.
+   */
+  async getSFASOverawards(
+    lastName: string,
+    birthDate: string,
+    sin: string,
+  ): Promise<SFASIndividual> {
+    const individual = await this.repo.findOne({
+      select: {
+        id: true,
+        cslOveraward: true,
+        bcslOveraward: true,
+      },
+      where: {
+        lastName: Raw((alias) => `LOWER(${alias}) = LOWER(:lastName)`, {
```

Comment:
I liked the solution using the `Raw` to workaround the object query limitations ❤️